packages/compass-components/src/components/document-list/document-actions-group.tsx

```diff
@@ -44,6 +44,13 @@ const actionsGroupSignalPopover = css({
   display: 'block !important',
 });
 
+const expandButton = css({
+  '& > div:has(svg)': {
```

Comment:
This is less ideal. I'd like to upstream a suggestion for a `Size.XSmallCompact` size [here](https://github.com/mongodb/leafygreen-ui/blob/9776f5f42d8e576bb9c22f215a9c5028ced9ff3f/packages/button/src/types.ts#L23) and [here](https://github.com/mongodb/leafygreen-ui/blob/9776f5f42d8e576bb9c22f215a9c5028ced9ff3f/packages/button/src/ButtonContent/ButtonContent.styles.ts#L58-L62).